src/java/org/apache/cassandra/service/accord/api/AccordAgent.java

```diff
@@ -182,29 +185,37 @@ public long attemptCoordinationDelay(Node node, SafeCommandStore safeStore, TxnI
         RoutingKey homeKey = command.route().homeKey();
         Shard shard = node.topology().forEpochIfKnown(homeKey, command.txnId().epoch());
 
+        // TODO (expected): make this a configurable calculation on normal request latencies (like ContentionStrategy)
         long oneSecond = SECONDS.toMicros(1L);
+        long startTime = mostRecentAttempt.hlc() + DatabaseDescriptor.getAccord().recover_delay.to(MICROSECONDS)
```

Comment:
I don't understand this, the `hlc` + the recovery delayConfig + `retryCount` == 0 then return 0 otherwise a random number?